### PR TITLE
Fix no background

### PR DESCRIPTION
**Pull request by mwestphal**
Fix https://github.com/f3d-app/f3d/issues/53

### CODE REVIEW

File: src/F3DLoader.cxx [hunk 1]
```diff
@@ -185,7 +185,7 @@ int F3DLoader::Start(int argc, char** argv, vtkImageData* image)
       else
       {
         retVal =
-          F3DOffscreenRender::RenderTesting(this->RenWin, this->Options.Reference, this->Options.RefThreshold, this->Options.Output) ?
+          F3DOffscreenRender::RenderTesting(this->RenWin, this->Options.Reference, this->Options.RefThreshold,  this->Options.NoBackground, this->Options.Output) ?
```
Comment (mwestphal) [inline src/F3DLoader.cxx:188]: double space

File: src/vtkF3DRenderPass.cxx [hunk 2]
```diff
@@ -254,8 +254,16 @@ void vtkF3DRenderPass::Blend(const vtkRenderState* s)
     {
       ssImpl << "  vec3 bgCol = texture(texBackground, texCoord).rgb;\n";
     }
-    ssImpl << "  vec3 result = bgCol * (1.0 - mainSample.a) + mainSample.rgb * mainSample.a;\n"
-              "  gl_FragData[0] = vec4(result.rgb, 1.0);\n";
+    ssImpl << "  vec3 result = bgCol * (1.0 - mainSample.a) + mainSample.rgb * mainSample.a;\n";
```
Comment (Meakk) [inline src/vtkF3DRenderPass.cxx:257]: using `mix` to interpolate would be cleaner imo: https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/mix.xhtml

**Review by Meakk — APPROVED** [review]
Feel free to ignore my comment, I know the code was already like this.

**Comment by mwestphal** [comment]
codecov reduction is small and expected, merging.

**Pull request merged**